### PR TITLE
Sandbags are no longer bulky, can hold in construction pouch and be build faster by engies

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Title

## Why It's Good For The Game

Sandbags are absolutly not convenient to use, this fixes it. Real combat engie one day?

## Changelog
:cl:
balance: Sandbags are no longer bulky, can hold in construction pouch and be build faster by engies
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
make sandbags bulky and make construction pouch bypass teh sandbag w class

File: code/game/objects/items/stacks/sandbags.dm [hunk 1]
```diff
@@ -105,8 +105,8 @@
 		return
 	user.visible_message("<span class='notice'>[user] starts assembling a sandbag barricade.</span>",
 	"<span class='notice'>You start assembling a sandbag barricade.</span>")
-
-	if(!do_after(user, 20, TRUE, src, BUSY_ICON_BUILD))
+	var/building_time = 2 SECONDS *(1 - clamp(usr.skills.getRating("construction") * 0.10, 0 , 0.50))
```
Comment (SplinterGP) [inline code/game/objects/items/stacks/sandbags.dm:108]:
```suggestion
	var/building_time = 2 SECONDS *(1 - clamp(user.skills.getRating("construction") * 0.10, 0 , 0.50))
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by Swagile** [comment]
> 
> 
> make sandbags bulky and make construction pouch bypass teh sandbag w class

this invalidates jaeger engi storage tho

engis already have storage issues that require jaeger engi storage if you actually play the class as intended (and not as a plasma cutter spammer)

**Comment by Chaznoodles** [comment]
Fun fact, you can already put filled sandbags in the jaeger engineering storage module 👀

File: code/game/objects/items/stacks/sandbags.dm [hunk 2]
```diff
@@ -105,8 +105,8 @@
 		return
 	user.visible_message("<span class='notice'>[user] starts assembling a sandbag barricade.</span>",
 	"<span class='notice'>You start assembling a sandbag barricade.</span>")
-
-	if(!do_after(user, 20, TRUE, src, BUSY_ICON_BUILD))
+	var/building_time = 2 SECONDS *(1 - clamp(user.skills.getRating("construction") * 0.10, 0 , 0.50))
```
Comment (SplinterGP) [inline code/game/objects/items/stacks/sandbags.dm:108]: use LERP here and replace fastest speed with low as can go and maximumskill with the point it reaches the maximum speed
```suggestion
	var/building_time = LERP(2 SECONDS, fastestspeed, user.skills.getPercent("construction", maximumskill)
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**